modules/ROOT/pages/index.adoc

```diff
@@ -20,17 +20,15 @@ A service mesh is an independent architecture layer, which is encapsulated in a
 
 The sidecar proxy enables you to bring your existing non-Mule microservices from the Kubernetes Cluster as APIs into Anypoint Platform. You can then manage these APIs within API Designer, API Manager, and Exchange. Additionally, you can monitor the health of your APIs using API analytics.
 
-A mixer, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.
-
-The mixer is architected on a general-purpose plug-in model, which provides flexibility in dealing with different infrastructure backend applications. These plug-in applications, known as adapters, enable the mixer to interface with the backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on.
+With Anypoint Service Mesh 1.1, the mixer component is removed and its functionality is performed by the Anypoint Service Mesh Envoy filter. With the Envoy filter, requests are directly sent to the adapter, thereby minimizing additional processing.
```

Comment:
Some corrections:
- The mixer has been removed by Istio, not by Anypoint Service Mesh. 
- It is not totally correct to say that the mixer functionality has been replaced with envoy filter...  I would say that the extension of the authorization mechanism is now implemented by leveraging Envoy Filters

Reply:
Done! Thanks!